packages/base-map/src/styled.ts

```diff
@@ -74,14 +74,11 @@ export const LayerSelector = styled.aside`
 
   .layers-list {
     background: rgba(255, 255, 255, 0.95);
-    border-radius: 0.5em;
-    box-shadow: 0px -1px 15px -3px rgba(0, 0, 0, 0.1);
-    display: flex;
-    flex-direction: column;
     list-style-type: none;
     padding: 1em;
     position: absolute;
     right: 0;
+    z-index: 1000;
 
     label {
```

Comment:
While we're in here, I'm wondering if we can add a `margin-bottom: 0` here and a `margin-bottom: 5px` on `&.fake-mobile-hover` at line 102? There's a small margin on every list item that's skewing the button: 

![image](https://user-images.githubusercontent.com/115499534/236945210-4a804f85-cba5-4a9a-a26e-fa7068fc47a2.png)

Reply:
I am aware of this glitch. That styling is set by OTP-RR, so I am looking to do a PR there to fix that.